tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -3891,8 +3894,20 @@ Status ConvertBiasAdd(OpConverterParams* params) {
   // If the input is NCHW, then we need to unsqueeze the bias such that its last
```

Comment:
The existing comment is only for the implicit batch mode. Let's update the comment to include explicit batch mode.

Reply:
Updated it.